### ISSUE TITLE
VScode.dev not matching up with GitHub repository. 

### ISSUE THREAD

**Issue by Apexofacircle** [microsoft/vscode#143959]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: Online
- OS Version: Chrome Os

Steps to Reproduce:

1. GO to Vscode.dev
2. Open remote site repository

The files in VScode.dev are not matching up with the files in my repository, This is the repository I am trying to access: https://github.com/Apexofacircle/apexofacircle.github.io

![Screenshot 2022-02-25 09 21 41](https://user-images.githubusercontent.com/96547482/155731176-efa042a0-9d98-4e08-b557-e45ae8329d82.png)
![Screenshot 2022-02-25 09 21 18](https://user-images.githubusercontent.com/96547482/155731177-a83b6614-47fb-4648-9839-4ac667a83286.png)

**Comment by joyceerhl** [comment]
@Apexofacircle, This happens whenever you have pending local changes that require a sync with GitHub. From the screenshot above, the index.html file has uncommitted changes, as indicated by the warning icon file decoration. 
![image](https://user-images.githubusercontent.com/30305945/155744501-e831b2e3-87e8-497d-830a-0214e7817317.png)


To sync repository contents, you can do any of the following:
1. commit your changes--we will prompt you to pull in the incoming changes as part of this process
2. discard your local changes if you don't want them anymore and just want to apply the incoming changes
3. click the sync button in the bottom left corner of the editor:
![image](https://user-images.githubusercontent.com/30305945/155744032-1ea43e00-fa62-437c-b4ab-cf7865681db5.png)

Related: https://github.com/microsoft/vscode-remote-repositories-github/issues/115

**Comment by Apexofacircle** [comment]
Thank you, I am an idiot. The refresh button worked. Sorry for wasting your time.

**Comment by joyceerhl** [comment]
No apologies needed, we have received similar feedback that this is confusing. Do you think the proposals in https://github.com/microsoft/vscode-remote-repositories-github/issues/115 help to reduce confusion about why vscode.dev gets out of sync with the contents on github.com?

**Comment by Apexofacircle** [comment]
A little bit.

**Comment by joyceerhl** [comment]
Thanks for the feedback. Closing as a duplicate of https://github.com/microsoft/vscode-remote-repositories-github/issues/115 since it sounds like you are unblocked for now and the underlying UX issue is being tracked there.
